src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php

```diff
@@ -64,7 +64,10 @@ public function testValidSignature(string $message)
 
         $time = time();
 
-        $lastResponse = new MockResponse();
+        $data = json_encode([
+            'ids' => ['0611223344'],
```

Comment:
@OskarStark @antiseptikk Do ids have a special format that would be worth to follow here?

Reply:
Hmm based on https://docs.ovh.com/gb/en/sms/send_sms_messages_via_url_-_http2sms

```json
{"status":100,"creditLeft":"1987","SmsIds":["10867690"]}
```

Reply:
But based on https://docs.ovh.com/gb/en/sms/send_sms_with_ovhcloud_api_in_php/

it is:
```json
Array
(
    [totalCreditsRemoved] => 1
    [invalidReceivers] => Array
        (
        )

    [ids] => Array
        (
            [0] => 26929925
        )

    [validReceivers] => Array
        (
            [0] => +3360000000
        )

)
```

which I think is correct